common/middleware/set-primary-navigation.js

```diff
@@ -39,6 +39,14 @@ function setPrimaryNavigation(req, res, next) {
     })
   }
 
+  if (permissions.check('moves:view:incoming', userPermissions)) {
+    items.push({
+      text: req.t('primary_navigation.incoming'),
+      href: '/moves/incoming',
+      active: req.path.startsWith('/moves') && req.path.endsWith('/incoming'),
+    })
+  }
+
```

Comment:
Not in scope for this PR, but maybe we could have an object keyed by permission and with the `item` as value, then iterate it to retrieve the wanted matches.

Reply:
I may actually just add this here as it feels like this is the point it could use a small refactor to reduce repetitiveness.

Reply:
something like

```js
const navPermissions = {
  'moves:view:outgoing': {
    text: 'primary_navigation',
    href: '/moves/outgoing',
    active: {
      startsWith: '/moves',
      endsWith: '/outgoing'
    }
  },
  'allocations': {
    text: 'primary_navigation',
    href: '/allocations',
    active: {
      startsWith: '/allocations',
      endsWith: '/outgoing'
    }
  }
  ...
}

Object.keys(navPermissions).forEach(permission => {
  if (permissions.check(permission, userPermissions)) {
    const { text, href, active = {} } = navPermissions[permission]
    const { startsWith, endsWith } = active
    items.push({
      text: req.t(text),
      href,
      active: (!startsWith || req.path.startsWith(startsWith)) && (!endsWith || req.path.endsWith(endsWith)),
    })
  }
})
```

Could a permission require more than one item to be added to the nav?

Reply:
Or indeed, what you've done in the !fixup.

Reply:
>Could a permission require more than one item to be added to the nav?

Not at this stage. Could also extend if it we need that in the future. Tried to go with the simplest cleanup for now.